examples/hacker_news/hacker_news/jobs/dbt_metrics.py

```diff
@@ -28,33 +27,25 @@ def dbt_metrics():
 
 dbt_prod_job = dbt_metrics.to_job(
     resource_defs={
-        **RESOURCES_PROD,
-        **{
-            "dbt": dbt_prod_resource,
-            # this is an alternative pattern to the configured() api. If you know that you won't want to
-            # further configure this resource per pipeline run, this can be a bit more convenient than
-            # defining an @resource with a config schema.
-            "dbt_assets": ResourceDefinition.hardcoded_resource(
-                SnowflakeQueryDbtAssetResource(
-                    {**{"database": "DEMO_DB"}, **SHARED_SNOWFLAKE_CONF}, "hackernews"
-                )
-            ),
-            "partition_bounds": ResourceDefinition.none_resource(),
-        },
+        "dbt": dbt_prod_resource,
+        # this is an alternative pattern to the configured() api. If you know that you won't want to
+        # further configure this resource per pipeline run, this can be a bit more convenient than
+        # defining an @resource with a config schema.
+        "dbt_assets": ResourceDefinition.hardcoded_resource(
+            SnowflakeQueryDbtAssetResource(
+                {**{"database": "DEMO_DB"}, **SHARED_SNOWFLAKE_CONF}, "hackernews"
+            )
+        ),
```

Comment:
In situations where the resource isn't re-used, it isn't clear to me whether you would prefer to set that resource on the job, or add it to the set provided to the repository. I chose to leave the existence and the prod/staging jobs here, but you could also imagine a world where these resources were subsumed into the prod/staging resource sets, and we got rid of the `dbt_staging` and `dbt_prod` jobs altogether.

Reply:
I can't necessarily speak universally, but if it was me, I'd want to put them on the repo so that I don't need to have multiple versions of the job.

Reply:
The unfortunate thing about this is keeping around `to_job`, and still needing to create 3 different versions of the same job. So maybe we would prefer to just throw the resources into the repo directly.